cdn/snippets.py

```diff
@@ -31,6 +31,7 @@
 
 
 # [START sign_url]
```

Comment:
Please remember to clean up old region tags after transitioning the docs to using the new ones.